app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootResolver.kt

```diff
@@ -57,6 +58,21 @@ internal object StorageRootResolver {
         return roots
     }
 
+    fun getFakeStorageRootForUri(context: Context, uri: Uri): SafOption {
+        return SafOption(
+            authority = AUTHORITY_STORAGE,
+            rootId = "fake",
+            documentId = "fake",
+            // TODO: Use something other than the USB icon?
+            icon = getIcon(context, AUTHORITY_STORAGE, "usb", 0),
+            title = context.getString(R.string.storage_user_selected_location_title),
+            summary = context.getString(R.string.storage_user_selected_location_summary),
+            availableBytes = null,
+            isUsb = false, // TODO: Check this if possible instead of forcing false
+            requiresNetwork = false, // TODO: Check this if possible instead of forcing false
```

Comment:
I guess those are the downsides of using a fake root. We can't know how to set those flags.

Reply:
Hmm. I guess we could ask the user if the storage they selected is USB to fix this? Could open a ticket for this if you think it would work...